Tasks/ConsoleApp/AsyncAwait/ProductsContext.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using Microsoft.EntityFrameworkCore;
+namespace AsyncAwait
+{
+    public class ProductsContext : DbContext
+    {
+        public DbSet<Product> _products { get; set; }
+
+        private string _connectionString { get; set; }
```

Comment:
Why is that property if it's private variable without interaction with external code?